tests/components/devolo_home_network/test_switch.py

```diff
@@ -132,6 +157,24 @@ async def test_update_enable_guest_wifi(hass: HomeAssistant, mock_device: MockDe
         assert state.state == STATE_ON
         turn_on.assert_called_once_with(True)
 
+    async_fire_time_changed(
+        hass, dt.utcnow() + timedelta(seconds=REQUEST_REFRESH_DEFAULT_COOLDOWN)
+    )
```

Comment:
Do we need to block till done after moving time?

Reply:
Yes indeed. I missed that.